wheel/src/run_generator.rs

```diff
@@ -28,8 +33,50 @@ pub struct PySpend {
     pub agg_sig_me: Vec<(Bytes48, Bytes)>,
 }
 
-#[pyclass(subclass, unsendable)]
-#[derive(Clone)]
+#[pymethods]
+impl PySpend {
+    #[new]
+    fn new(
+        coin_id: Bytes32,
+        puzzle_hash: Bytes32,
+        height_relative: Option<u32>,
+        seconds_relative: u64,
+        create_coin: Vec<(Bytes32, u64, Option<Bytes>)>,
+        agg_sig_me: Vec<(Bytes48, Bytes)>,
+    ) -> PySpend {
+        PySpend {
+            coin_id,
+            puzzle_hash,
+            height_relative,
+            seconds_relative,
+            create_coin,
+            agg_sig_me,
+        }
+    }
+}
+
+#[pyproto]
+impl PyObjectProtocol for PySpend {
```

Comment:
this is duplicated twice, and I suspect will be duplicated more times going forward. I'm kind of surprised pyo3 doesn't have a macro to enable the obvious `__str__` and `__repr__` implementations as well as the comparisons.

I considered writing a derive macro for this, but as I understand it, that would have to live in a separate crate, which seemed like a steep hurdle right now.